toolkits/global/packages/global-forms/README.md

```diff
@@ -69,68 +85,125 @@ This component supports a wide range of standard form field attributes. For exam
+There are 2 ways to add fieldsets to your form:
+1. Create a field with the template value of `globalFormFieldset` and an array of fields using its `fields` property.
+2. Create an array of fieldsets and loop over them, assigning the globalFormFieldset partial.
+
+#### (1) The fieldset field
 
 ```json
-"hint": "This will be the email address you used when registering"
+{
+    "template": "globalFormFieldset",
+    "legend": "<h1>Login</h1>",
+    "fields": [
+        ...
+    ]
 ```
+    
+#### (2) A fieldset array
+In this case, each fieldset is an object:
 
-#### Required
+```json
+"myFieldsets": [
+    {
+        "legend": "<h2>One fieldset</h2>",
+        "fields": [
+            ...
+        ]
+    },
+    {
+        "legend": "<h2>Another, different fieldset</h2>",
+        "fields": [
+            ...
+        ]
+    }    
+]
+```
 
-Fields are required by default. Avoid asking for information you don't really need. To explicitly mark fields as required, `requiredSuffix` will append text—such as an asterisk—to the label. This property permits HTML, meaning you can hide the asterisk character from screen readers.
+And the accompanying template would iterate over the fieldsets, applying the `globalFormFieldset` partial:
 
-```
-"requiredSuffix": "<span aria-hidden=\"true\">*</span>"
+```html
+{{#each myFieldsets}}
+    {{> globalFormFieldset}}
+{{/each}}
 ```
 
-The `required` and `aria-required="true"` attributes are applied automatically. Unless `novalidate: true` is applied on the field’s data, browser validation will warn the user when they have not filled out the field upon submission.
 
-If you do need to ask for optional information, set `optional: true`. This will remove `required` and `aria-required="true"` as well as appending the text “(optional)” to the field label. 
+## Legends
 
-#### Data properties
+Use HTML to style the legend and add semantic meaning. In most cases, legends should be headings, with the heading level determined by the page structure.
 
-In addition to these top-level properties, you can add data properties as a `dataAttrs` array, which can be useful for unit testing.
+![Legends](https://user-images.githubusercontent.com/15365576/210399801-a3078572-c8cb-4826-93bb-02b2aabb9fbc.png)
 
 ```json
-{
-    "template": "globalFormText",
-    "label": "Your email address",
-    "id": "your-email",
-    "name": "your-email",
-    "dataAttrs": {
-        "test": "someValue"
-    }
+"fieldsets": {
+    [
+        {
+            "legend": "<h2>My level 2 legend</h2>",
+            "fields": [
+                ...
+            ]
+        }
+    ]
 }
 ```
 
-This would create the following attribution: `data-test: someValue`.
+For accessibility, do not use a fieldset without an accompanying legend. If a legend is not appropriate, neither is a fieldset; just use fields without a fieldset parent.
```

Comment:
Not sure we really need 'For accessibility' here - should either remove or be more specific about the reason for it.

Reply:
@Heydon Is this to do with screenreaders?

Reply:
@foxintherain @alexhallspringer  Largely, yes. Happy to remove "For accessibility..." or—if we want to be more specific—we can have:

_"Fieldsets represent groups of functionality and groups need to be labeled, with a legend. If a legend is not appropriate, neither is a fieldset; just use fields without a fieldset parent."_

Reply:
```suggestion
Do not use a fieldset without an accompanying legend. If a legend is not appropriate, neither is a fieldset; just use fields without a fieldset parent.
```